Sources/SFSafeSymbols/Symbols/SFSymbol+AllSymbols.swift

```diff
@@ -6,20 +6,31 @@ extension SFSymbol {
     public static var allCases: [SFSymbol] { Array(allSymbols) }
 
     public static var allSymbols: Set<SFSymbol> = {
+        var result = allSymbols1_0
+        if #available(iOS 14.0, tvOS 14.0, watchOS 7.0, *) {
+            result.formUnion(allSymbols2_0)
```

Comment:
[Optional] I would suggest to use another naming: `symbolsAvailableSince2_0`. It's an `internal` variable and one could argue that naming is not so important, but `allSymbols2_0` is not the proper description of what's behind the variable.